atom-publisher-lib/src/main/scala/com/gu/atom/util/JsonSupport.scala

```diff
@@ -66,7 +66,8 @@ object JsonSupport {
         val fixedJson = {
           val nestedDataFieldName = atomType.toLowerCase
 
-          if (data(nestedDataFieldName).nonEmpty) c.value
+          //Special case for commonsDivision because the scrooge enum value loses the casing
+          if (data(nestedDataFieldName).nonEmpty || nestedDataFieldName == "commonsdivision") c.value
```

Comment:
Scrooge 😠 🔥 🔫

Reply:
I don't get it. Since `val nestedDataFieldName = atomType.toLowerCase` it should already be all lowercase, no?

Reply:
```scala
if (data(nestedDataFieldName).nonEmpty || nestedDataFieldName == "commonsdivision" && data("commonsDivision").nonEmpty)
```
?

Reply:
The commons division atom data will always be in the new (correct) format, so we always want to return `c.value` here.
The `else` branch is for compatibility with some older entries where the `data` object doesn't specify the actual type of the atom data. Ideally we wouldn't have to use the `atomType` enum field to determine the type of the `data` union field.

Reply:
Soon this bull*** won't be necessary anymore:
https://github.com/regiskuckaertz/humbug/blob/master/src/test/fixtures/scala/contentatoms/AtomType.scala
https://github.com/regiskuckaertz/humbug/blob/master/src/test/fixtures/scala/contentatoms/Atom.scala
https://github.com/regiskuckaertz/humbug/blob/master/src/test/fixtures/scala/contentatoms/AtomData.scala
😎